### ISSUE TITLE
𝗣𝗮𝗴𝗲: Create Individual Listings Page

### ISSUE THREAD

**Issue by CaelumD25** [UVicMartletplace/martletplace#72]
## Feature Description

**View Individual Listing: Users should be able to open a listing and view the product card displaying attributes like images, title, description and current status.**
This page should be what is presented to users whenever they click on a listing summary card(see #48). It should at least include a title, description, price, status, and a image(can be placeholder for now). Below this listing reviews and review composition will eventually be available below the listing, however, it is not required for this sprint so that is of extremely low priority. This requires components of:
- #55 
- #51 

*Note: While there is a mocked page indicating the status of an item, it was rushed and I think we can do better to show that the item has been sold*

relates: #55, #51, #53, #52, #79 
blocked by: #55, #51

## Problem You're Trying to Solve

[See SRS for User Story](https://docs.google.com/document/d/1DNqexMBf4MpYPCxWsT3plUDAjDSbdJwUzw-UyZX-vDs/edit#heading=h.wjfgcg464hmc)

## Related External Resources

See [Get Listings](http://market.lebl.ca/openapi/#/Listings/getListing)
 and [The Listing Object](http://market.lebl.ca/openapi/#/:~:text=NewReview-,Listing,-NewListing) on the OpenAPI spec

![Image](https://github.com/UVicMartletplace/martletplace/assets/54737724/aacc3381-f5d2-49c4-a0ee-ab041c9c7617)
![Image](https://github.com/UVicMartletplace/martletplace/assets/54737724/40c47c5f-5501-4293-8912-2a1d1c5f0c03)
